packages/cli/lib/prompts/sandboxesPrompt.js

```diff
@@ -50,6 +50,26 @@ const sandboxNamePrompt = () => {
   ]);
 };
 
+const developmentSandboxNamePrompt = () => {
+  return promptUser([
+    {
+      name: 'name',
+      message: i18n(`${i18nKey}.name.developmentSandboxMessage`),
+      validate(val) {
+        if (typeof val !== 'string') {
+          return i18n(`${i18nKey}.name.errors.invalidName`);
+        } else if (!val.length) {
+          return i18n(`${i18nKey}.name.errors.nameRequired`);
+        }
+        return accountNameExistsInConfig(val)
+          ? i18n(`${i18nKey}.name.errors.accountNameExists`, { name: val })
+          : true;
+      },
+      default: 'New development sandbox',
+    },
+  ]);
+};
+
```

Comment:
This looks similar enough to the original `sandboxNamePrompt` that you could just pass a `isDevelopment` arg to it or something rather than making it its own function